src/Asset.cpp

```diff
@@ -10,7 +10,9 @@ namespace fs = std::filesystem;
 std::unique_ptr<Asset> Asset::instance;
 
 
-#if defined(__APPLE__) || defined(__MACH__) || defined(__linux__)
+#if defined(__APPLE__) || defined(__MACH__)
+uint32_t path_length = PATH_MAX;
```

Comment:
Could be a constexpr :P

Reply:
Didn't work for me

Reply:
Oh, that is weird. Is PATH_MAX not a constant value?